### PR TITLE
Add new term entry for javascript set date method

### LINKED ISSUE #2428: [Term Entry] JavaScript Dates .setDate()
### Reason/inspiration (optional)

We would like a new term entry for JavaScript [Dates](https://www.codecademy.com/resources/docs/javascript/dates) about the `.setDate()` method. The entry should go in a new file in `docs/content/javascript/concepts/dates/terms/setDate/setDate.md`

The entry should
    - Have an introductory paragraph about what the method is and does.
    - An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cguttweb**
### Description

Create a new term entry for the JavaScript setDate method resolves #2428 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
Looks good to me! :)

File: content/javascript/concepts/dates/terms/setDate/setDate.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Title: '.setDate()'
+Description: 'Changes the day of the month according to local time.'
+Subjects: 
+  - 'Web Development'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent: 
+  - 'intro-to-javascript'
+  - 'path/front-end-engineer'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setDate/setDate.md:11]:
```suggestion
  - 'paths/front-end-engineer-career-path'
```

**Comment by cguttweb** [comment]
@caupolicandiaz I've made updates based on feedback thanks.

**Review by caupolicandiaz — APPROVED** [review]
@cguttweb Overall, looks good. Just made a couple of minor additions.

**Comment by yangc95** [comment]
🙌 @cguttweb live link: https://www.codecademy.com/resources/docs/javascript/dates/setDate

Screenshot 2023-06-26 at 3 38 09 PM

**Pull request merged**